### ISSUE TITLE
Upload the infographic to Multimedia page

### ISSUE THREAD

**Issue by MarijaIgnj** [AccessibleAI-edu/accessibleai-edu#35]
[Generative AI tools](https://drive.google.com/file/d/1ZclFreWBiRCOnRH8CA0tgUZR3y6jxQMX/view?usp=sharing)

When you click a specific item in the Multimedia, it should open the page which contains:

1. the infographic embedded in the page
2. a PDF button to download the infographic

**Comment by MarijaIgnj** [comment]
@mutharasus Read the instructions above.

**Comment by mutharasus** [comment]
@MarijaIgnj can you please clarify what the requirement is. So when we click on the specific item you would like to see a page that has the infographic in a PDF format? Or do you want to see it as an image. If you would like to see it as an image how would the multiple pages look? Even a pencil drawing of what you would like to see in the screen would be sufficient. Thanks.

**Comment by MarijaIgnj** [comment]
@mutharasus I like this solution. Can the Generative AI tools infographic be in one line, like the other one (not divided into two pages)?

**Comment by MarijaIgnj** [comment]
@mutharasus Yes, that is fine, looks good, I suggested it in the document as well.

**Comment by MarijaIgnj** [comment]
@mutharasus Looks good now. Little thing, since this infographic is dark, can we make a thin white frame around it? If it is not possible, I can make it in Canva again and send you the new PDF to upload.
